internal/servers/controller/common/common.go

```diff
@@ -11,12 +12,13 @@ import (
 )
 
 type (
-	AuthTokenRepoFactory    = oidc.AuthTokenRepoFactory
-	IamRepoFactory          func() (*iam.Repository, error)
-	OidcAuthRepoFactory     = oidc.OidcRepoFactory
-	PasswordAuthRepoFactory func() (*password.Repository, error)
-	ServersRepoFactory      func() (*servers.Repository, error)
-	StaticRepoFactory       func() (*static.Repository, error)
-	SessionRepoFactory      func() (*session.Repository, error)
-	TargetRepoFactory       func() (*target.Repository, error)
+	AuthTokenRepoFactory       = oidc.AuthTokenRepoFactory
+	VaultCredentialRepoFactory func() (*vault.Repository, error)
```

Comment:
Can you add this definition as it is done above for `oidc.AuthTokenRepoFactory` I will need the repo factory within the vault package for the renewal job work I am doing